main.js

```diff
@@ -1 +1,10 @@
-
+function toggleToDo(element){
+  var newClass;
```

Comment:
@ChrisBaidoo, let's use `const` instead of `var` in this project

Reply:
@samdemaeyer using "const" makes it difficult to change variable later. I have used "let" instead.

Reply:
Good catch @ChrisBaidoo! `const` is indeed not a valid useage in this case, only `var` or `let` can be used. So let’s go for `let`